eng/SourceBuild.props

```diff
@@ -4,8 +4,21 @@
     <GitHubRepositoryName>roslyn</GitHubRepositoryName>
     <SourceBuildManagedOnly>true</SourceBuildManagedOnly>
     <SourceBuildTrimNetFrameworkTargets>true</SourceBuildTrimNetFrameworkTargets>
+    <!-- Roslyn produces stable release branded and stable pre-release branded artifacts in addition to the normal non-stable artifacts.
+         Only the non-stable artifacts should flow downstream in source build -->
+    <EnableDefaultSourceBuildIntermediateItems>false</EnableDefaultSourceBuildIntermediateItems>
   </PropertyGroup>
 
+  <Target Name="GetCustomIntermediateNupkgContents" BeforeTargets="GetCategorizedIntermediateNupkgContents">
+    <ItemGroup>
+      <IntermediateNupkgArtifactFile Include="$(CurrentRepoSourceBuildArtifactsPackagesDir)Shipping\**\*.nupkg" />
+      <IntermediateNupkgArtifactFile Include="$(CurrentRepoSourceBuildArtifactsPackagesDir)NonShipping\**\*.nupkg" />
+      <!-- Don't pack any symbol packages: not needed for downstream source-build CI.
```

Comment:
How does this stop us from packing?

Reply:
https://github.com/dotnet/roslyn/pull/67679/commits/c6b058c059250b213fe204e4e46e17427403f772#diff-6a28d92de8923be8821f2123541dc23c8ad0359c96117709e6d49e752c0303d2R9

This prevents the defaults (whole packages dir) from being included.